services/core-web/src/components/common/CoreTableCommonColumns.tsx

```diff
@@ -111,18 +148,46 @@ export const renderDocumentLinkColumn = (
 export const renderTaggedColumn = (
   dataIndex: string,
   title: string,
-  tag: string,
+  tag = "",
   sortable = false,
   placeHolder = ""
 ) => {
   return {
     title,
     dataIndex,
     key: dataIndex,
-    render: (text: any) => {
-      const content = <div title={title}>{text ?? placeHolder}</div>;
-      return withTag(tag, content);
+    render: (text: any, record: any) => {
+      const content = (
+        <div
+          className={record.number_of_versions !== undefined ? "file-name-text" : ""}
+          style={record?.number_of_versions === 0 ? { marginLeft: "38px" } : {}}
+        >
+          {text ?? placeHolder}
+        </div>
+      );
+      return tag !== "" || record.number_of_versions !== undefined
+        ? documentWithTag(record, content, title)
+        : withTag(tag, content);
     },
     ...(sortable ? { sorter: nullableStringSorter(dataIndex) } : null),
   };
 };
+
+export const documentActionOperationsColumn = (title = "", dataIndex = "operations") => {
+  return {
+    title,
+    dataIndex,
+    key: dataIndex,
+    render: (text: any, record: any) => {
```

Comment:
Does this need a dataIndex- are we expecting the records to have an "operations" attribute that we are using to index into the data?
If not, I'd recommend taking out dataIndex from the return, and change the render to `render: (record: any) => ...`

Reply:
Thanks for catching this, your right I don't think we will need this, this has been removed now.